artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/server/impl/JMSServerManagerImpl.java

```diff
@@ -1719,11 +1720,11 @@ public void callback(SimpleString queueName) throws Exception {
       @Override
       public void callback(SimpleString address, SimpleString queueName) throws Exception {
          Queue queue = server.locateQueue(address);
-         Collection<Binding> bindings = server.getPostOffice().getBindingsForAddress(address).getBindings();
```

Comment:
Is there a master counterpart for this fix?

Reply:
This callback doesn't exist on master so there's no direct counterpart for this fix. However, I do plan on reviewing all the usages of `getBindingsForAddress` to try to find similar problems.

Reply:
I opened #2469 as a result of my investigation on the master branch.